specification/synapse/resource-manager/Microsoft.Synapse/stable/2020-12-01/library.json

```diff
@@ -0,0 +1,123 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-12-01",
+    "title": "SynapseManagementClient",
+    "description": "Azure Synapse Analytics APIs"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "type": "oauth2",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      },
+      "description": "Azure Active Directory OAuth2 Flow"
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Synapse/workspaces/{workspaceName}/libraries": {
```

Comment:
If a user can list them they need to be able to GET a single library as well

Reply:
Currently, PUT and GET library operations are part of data plane APIs. For the preview release we are only exposing List operation in control plane to clients as that is the only API required for the preview feature. We would add GET library operation in the next release. Please let us know if there are any concerns.

Reply:
The resource provider contract requires that resources exposed via a GET collection have individual GETs.  If I have a resource ID I should be able to GET that ID, not required to LIST its type and search through the list.

Reply:
Got it. We will add GET API to swagger and code.